doc/introduction.rst

```diff
@@ -89,8 +85,72 @@ Why is machine learning relevant to NeuroImaging? A few examples!
 Installing nilearn
 ====================
 
-.. raw:: html
-   :file: install_doc_component.html
+.. tab-set::
+
+  .. tab-item:: Pip install
+
+      **1. Download and install 64-bit Anaconda**
+
+      We recommend that you install a complete
+      64-bit scientific Python distribution like
+      `Anaconda <https://www.anaconda.com/download/>`_ (or `PythonXY <http://python-xy.github.io/>`_).
+
+      Nilearn requires a Python installation and the following
+      dependencies: ipython, scipy, scikit-learn, joblib, matplotlib,
+      nibabel and pandas.
+
+      Once conda is installed, create and activte
+      a new python environment with:
+
+      .. code-block::
+
+         conda create -n nilearn-test python=3.9
+         conda activate nilearn-test
+
+      **2. Install nilearn with pip**
+
+      Execute the following command in the command prompt / terminal
+      in the proper conda environment:
+
+      .. code-block::
+
+         python -m pip install -U --user nilearn
```

Comment:
why `--user` if we are in a virtual env?

Reply:
I guess this was a relic from the past. I got rid of it!